src/templates/partners/user_testing.html.tt

```diff
@@ -12,24 +12,7 @@
         [% l("Register to become a tester using the links below:") %]
```

Comment:
@ashkanx I think we should change `links` to `link` since now there is only one link to choose